script/demoScripts/demoStargateV2Tx.ts

```diff
@@ -295,20 +293,20 @@ function transformQuoteOFTResponse(response: any[]): QuoteOFTResponse {
 }
 
 // This endpoint returns all tokens, their pools/routers and assetIds
-async function getSupportedTokensAndPools() {
-  const resp = await fetch(
-    'https://d3k4i7b673n27r.cloudfront.net/v1/metadata?version=v2'
-  )
-  const responseJson = await resp.json()
+// async function getSupportedTokensAndPools() {
```

Comment:
why is this commented out?

Reply:
no, all good. I like that rule that we have to justify when leaving commented code in PR